package/CHANGELOG

```diff
@@ -82,6 +82,8 @@ Enhancements
    (Issue #4673) 
  * enables parallelization for analysis.dssp.dssp.DSSP (Issue #4674)
  * Enables parallelization for analysis.hydrogenbonds.hbond_analysis.HydrogenBondAnalysis (Issue #4664)
+ * Added pytest for case where backend instance and AnalysisBase.run() have both different
```

Comment:
May I ask that we not add changelog entries for singular test additions. The changelog is really meant to be user facing "something has changed in the code". Large test changes are ok, but single test changes will clog up the changelog.

Reply:
Yup, adjusted it and removed the line from the Changelog :)